field_rrule/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Repetition rules
+================
+
+This module was written to offer a field type that holds rrules according to RFC 2445.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* depend on it
+* say ``from openerp.addons.field_rrule import FieldRRule``
+* use ``FieldRRule`` like any other field
+* on forms, use ``widget="rrule"``
+* have a look at `demo/res_partner.*`
+
+Technically, this is a wrapper around serialized fields. The value always will be a subclass of dateutil's ``rruleset``. For technical reasons, this class overrides ``__iter__``, so if you need a proper ``rruleset``, call the value: ``my_browse_record.my_field_of_type_rrule()`` - this gives you a vanilla ``rruleset``.
+
+If you want to pass a default, use the internal representation you'll find in the database - a list of dictionaries with the keyword arguments to be passed to rrule's constructor and a `type` field that for now can only be `rrule`: A context of ``{"default_rrule": [{"count": 1, "freq": 1, "type": "rrule", "interval": 1, "bymonthday": [1]}]}`` would give you a default for the field ``rrule`` which occurs one time at the first of the month.
```

Comment:
it does: https://github.com/OCA/server-tools/pull/480/files#diff-04e747b1caf1bcf4b384e6a3c4b8258aR64, but that's not going to help in a form definition or the xml describing an action